cases/invalid-k8s-mangled-yaml/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      lables:
+        app: nginx
+    speck: 
```

Comment:
this might be a bit too mangled.  Let's not include typos but use a pretty normal misconfiguration (eg misformatting a secretKeyRef in an env var)